src/client.js

```diff
@@ -65,10 +67,16 @@ export default class DockerExecWebsocketClient extends EventEmitter {
     this.strbuf.on('data', (data) => {
       this.outstandingBytes += data.length;
       debug(this.outstandingBytes);
-      this.socket.send(data, {binary: true}, () => {
+      if (BROWSER) {
+        this.socket.send(data, {binary: true}, () => {
+          this.outstandingBytes -= data.length;
+          debug(this.outstandingBytes);
+        });
+      } else {
+        this.socket.send(data);
         this.outstandingBytes -= data.length;
```

Comment:
So in browser environments we have no way to control how much data is currently in transmissions...

Hmm... Given that long term plans probably involves refactoring congestion control I think this acceptable :)